resources/templates/k8s/job.yml

```diff
@@ -27,6 +27,7 @@ spec:
       annotations:
         {{- if .ttl }}
         pod-complete.stage.kwok.x-k8s.io/delay: "{{.ttl}}"
+        pod-complete.stage.kwok.x-k8s.io/jitter-delay: "{{.ttl}}"
```

Comment:
does `jitter-delay` has to be equal to `delay`?

Reply:
No, it can be any value that is greater than or equal to the delay. The actual delay will be a random value between the delay (min) and jitter-delay (max). Setting them equal will result in a deterministic delay.